frontend/src/components/user/SignUpForm.js

```diff
@@ -37,13 +58,39 @@ const SignUpForm = ({ navigate }) => {
   const handlePasswordChange = (event) => {
     setPassword(event.target.value);
   }
+  
+  // double password input
+  const handleComparePasswords = (event) => {
+    setPasswordConfirmation(event.target.value)
+    }
+  
 
 // render of html object
     return (
       <form onSubmit={handleSubmit}>
-          <input placeholder="Username" id="username" type='text' value={ username } onChange={handleUsernameChange} />
-          <input placeholder="Email" id="email" type='text' value={ email } onChange={handleEmailChange} />
-          <input placeholder="Password" id="password" type='password' value={ password } onChange={handlePasswordChange} />
+          <input placeholder="Username"
+                id="username"
+                type='text'
+                value={ username }
+                onChange={handleUsernameChange} />
+          
+          <input placeholder="Email"
+                id="email" type='text'
+                value={ email }
+                onChange={handleEmailChange} />
+          
+          <input placeholder="Password"
+                  id="password"
+                  type='password'
+                  value={ password }
+                  onChange={handlePasswordChange}/>
+
+          <input placeholder="Password_confirmation"
+                  id="password_confirmation"
+                  type='password'
+                  value={ password_confirmation }
```

Comment:
Not a big deal at all but, dont like the underscore in the name lol. Just change it to "Reenter password" or something